src/Device/Driver/AirControlDisplay.cpp

```diff
@@ -123,6 +127,226 @@ ACDDevice::ParseNMEA(const char *_line, NMEAInfo &info)
     return false;
 }
 
+static void
```

Comment:
Much of this is copy'n'pasted from somewhere else. Duplicate code is not acceptable.

Reply:
I agree, duplicate code makes everything a mess. Some of the code comes from another driver. As far as I understand this, I cannot use methods from another driver (which also rules out using NMEAOut to forward the data). Since each driver has to be maintained to what the device it is designed for can do, it might also be not practical to merge code and implement the methods. FLARM for example is an exception to that of course. 

Ideas for a way forward?